ui/raidboss/data/05-shb/raid/e8n.txt

```diff
@@ -4,4 +4,105 @@
+131.2 "Mirror, Mirror" sync /:Shiva:4DD4:/
+143.4 "Driving Frost/Biting Frost" sync /:Shiva:4DD[BC]:/
+148.5 "Reflected Frost" sync /:Frozen Mirror:4DF[EF]:/
+160.6 "Redress" sync /:Shiva:4F4F:/
+165.7 "Shining Armor" sync /:Shiva:4DF1:/
+173.7 "Scythe Kick" sync /:Shiva:4DE3:/
+181.9 "--Untargetable--"
+193.2 "Shattered World" sync /:Shiva:4DE9:/
+
+# Intermission. Fixed length.
+209.4 "Heart Asunder" sync /:Mothercrystal:4E12:/
+218.1 "Stoneskin" sync /:Earthen Aether:4DEF:/
+223.5 "Heart Asunder" sync /:Mothercrystal:4E12:/
+228.2 "Shock Spikes" sync /:Electric Aether:4DEE:/
+233.2 "Rush" sync /:Luminous Aether:4FC8:/
+233.3 "Rush" sync /:Luminous Aether:4FC9:/
+237.5 "Heart Asunder" sync /:Mothercrystal:4E12:/
+251.5 "Heart Asunder" sync /:Mothercrystal:4E12:/
+256.2 "Shock Spikes" sync /:Electric Aether:4DEE:/
+260.2 "Stoneskin" sync /:Earthen Aether:4DEF:/
+261.2 "Rush" sync /:Luminous Aether:4FC8:/
+261.3 "Rush" sync /:Luminous Aether:4FC9:/
+
+# Post-intermission block
+272.6 "Skyfall" sync /:Shiva:4E13:/
+287.9 "--targetable--"
+306.9 "Holy" sync /:Shiva:4DE[CD]:/
+321.4 "Light Rampant" sync /:Shiva:4DE8:/
+329.6 "the Path of Light" sync /:Shiva:4DD9:/
+332.6 "Bright Hunger" sync /:Shiva:4E0C:/
+337.7 "the Path of Light" sync /:Shiva:4DD9:/
+340.6 "Bright Hunger" sync /:Shiva:4E0C:/
+345.9 "the Path of Light" sync /:Shiva:4DD9:/
+348.6 "Bright Hunger" sync /:Shiva:4E0C:/
+356.0 "Mirror, Mirror" sync /:Shiva:4DD4:/
+368.1 "Kick/Frost" sync /:Shiva:4D(?:D[BC]|E[23]):/
+373.2 "Reflected Kick/Frost" sync /:Frozen Mirror:(?:4DF[EF]|4E0[01]):/
+
+# Rotation block
+385.3 "Redress" sync /:Shiva:4F4E:/
+390.4 "Frost Armor" sync /:Shiva:4DF0:/
+398.4 "Twin Stillness/Twin Silence" sync /:Shiva:4DD[DE]:/
+413.7 "Double Slap" sync /:Shiva:4DDA:/
+424.8 "Mirror, Mirror" sync /:Shiva:4DD4:/
+437.0 "Kick/Frost" sync /:Shiva:4D(?:D[BC]|E[23]):/
+442.1 "Reflected Kick/Frost" sync /:Frozen Mirror:(?:4DF[EF]|4E0[01]):/
+455.2 "Absolute Zero" sync /:Shiva:4DD7:/
+467.4 "Redress" sync /:Shiva:4F4F:/
+472.6 "Shining Armor" sync /:Shiva:4DF1:/
+480.6 "Spiteful Dance/Embittered Dance" sync /:Shiva:4DE[4567]:/
+484.7 "Spiteful Dance/Embittered Dance" sync /:Shiva:4DE[4567]:/
+500.8 "Holy" sync /:Shiva:4DE[CD]:/
+513.9 "Absolute Zero" sync /:Shiva:4DD7:/
+
+526.1 "Redress" sync /:Shiva:4F4E:/
+531.2 "Frost Armor" sync /:Shiva:4DF0:/
+539.2 "Twin Stillness/Twin Silence" sync /:Shiva:4DD[DE]:/
+554.5 "Double Slap" sync /:Shiva:4DDA:/
+565.6 "Mirror, Mirror" sync /:Shiva:4DD4:/
+577.8 "Kick/Frost" sync /:Shiva:4D(?:D[BC]|E[23]):/
+582.9 "Reflected Kick/Frost" sync /:Frozen Mirror:(?:4DF[EF]|4E0[01]):/
+596.0 "Absolute Zero" sync /:Shiva:4DD7:/
+608.2 "Redress" sync /:Shiva:4F4F:/
+613.4 "Shining Armor" sync /:Shiva:4DF1:/
+621.4 "Spiteful Dance/Embittered Dance" sync /:Shiva:4DE[4567]:/
+625.5 "Spiteful Dance/Embittered Dance" sync /:Shiva:4DE[4567]:/
+641.6 "Holy" sync /:Shiva:4DE[CD]:/
+654.7 "Absolute Zero" sync /:Shiva:4DD7:/
+
+666.9 "Redress"
```

Comment:
Is there supposed to be a jump here?

Reply:
There is! I even remember adding it early on. Wherever did it disappear...